README.md

```diff
@@ -1,13 +1,18 @@
 # IO
 
-Go packages for pheripheral I/O. It contains driver for the following chips:
+Go packages for pheripheral I/O. It contains driver for the following IC's:
 
 * SPI
     * [Microchip](spi/microchip)
```

Comment:
I think is it's more sophisticated to reference to online godoc instead of documents in repo.